src/_gettsim/transfers/arbeitslosengeld_2/regelbedarf.py

```diff
@@ -0,0 +1,509 @@
+"""Basic needs following SGB II."""
+
+from _gettsim.functions.policy_function import policy_function
+
+
+@policy_function()
+def regelbedarf_m(
+    regelsatz_m: float,
+    kosten_unterkunft_m: float,
```

Comment:
```suggestion
    kosten_der_unterkunft_m: float,
```
This is a set phrase.